pkg/client/client.go

```diff
@@ -36,6 +37,8 @@ type ParagliderControllerClient interface {
 	DeletePermitListRules(namespace string, cloud string, resourceName string, rules []string) error
 	CreateResource(namespace string, cloud string, resourceName string, resource *paragliderpb.ResourceDescriptionString) (map[string]string, error)
 	AttachResource(namespace string, cloud string, resource *orchestrator.ResourceID) (map[string]string, error)
+	CheckResource(namespace string, cloud string, resourceName string) (map[int32]string, error)
```

Comment:
For the client, we're currently maintaining 2 different functions for check and fix, since they both use different request methods. On the orchestrator side though, we do maintain just one function, passing in a boolean to apply fix if possible. Down the line, maybe we could combine check and fix into one function, using a POST method (Should be fine since not all post requests are guaranteed to change the state of the system any way)